core/src/main/web/outputdisplay/bko-latex.js

```diff
@@ -24,7 +24,11 @@
     return {
       link: function(scope, element, attrs) {
         scope.$watch('model.getCellModel()', function(newValue) {
-          katex.render(newValue, element[0]);
+          try {
```

Comment:
something to keep in mind as soon as we use a try catch the entire code block gets deoptimized in v8. Since this is inside a `$watch` this might actually add up perf wise sine it is going to get hit during every digest loop.

This might be worth investigating if there is another way other than a try catch.